src/App.jsx

```diff
@@ -9,32 +12,27 @@ export const tabs = [
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">
-      Selected tab is Tab 1
-    </h1>
-
-    <div data-cy="TabsComponent">
-      <div className="tabs is-boxed">
-        <ul>
-          <li className="is-active" data-cy="Tab">
-            <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-          </li>
-
-          <li data-cy="Tab">
-            <a href="#tab-2" data-cy="TabLink">Tab 2</a>
-          </li>
+export const App = () => {
+  const [selectebTadId, setSelectedTadId] = useState(tabs);
+  const [state, setState] = useState('tab-1');
 
-          <li data-cy="Tab">
-            <a href="#tab-3" data-cy="TabLink">Tab 3</a>
-          </li>
-        </ul>
-      </div>
+  return (
+    <div className="section">
+      <h1 className="title">
+        {selectebTadId.map(tab => (
+          `${state === tab.id ? `Selected tab is ${tab.title}` : ''}`
```

Comment:
don't see any sense to use a ternary operator here 
```suggestion
          {state === tab.id && `Selected tab is ${tab.title}`}
```

Reply:
Так не працює